builder.config.js

```diff
@@ -23,8 +23,8 @@ const config = {
     output: 'dist/main'
   },
   win: {
-    target: 'NSIS',
-    artifactName: 'Installar_Mapeo_v${version}_${os}-${env.ARCH}.${ext}',
+    target: [{ target: 'nsis', arch: ['x64', 'ia32'] }],
+    artifactName: 'Installar_Mapeo_v${version}_${os}-${arch}.${ext}',
```

Comment:
since the new version of `electron-builder` actually handles multiple arches properly, we can use its built-in `arch` variable here instead